docs/usage.md

```diff
@@ -0,0 +1,40 @@
+# Using Config Sync
```

Comment:
Suggest add a link to this doc in README as well

Reply:
done